_includes/donateButton.html

```diff
@@ -0,0 +1,5 @@
+<a href="https://www.paypal.com/donate/?hosted_button_id=RH4Y34HGQ535S" target="_blank">
+    <button>
+        Donate
+    </button>
```

Comment:
It’s fairly unusual to nest a `<button>` inside a link. We’re at the mercy of the user’s operating system, which will most likely put the text against a light background, even though the link text is also light, resulting in inadequate contrast:

![White-on-white donate button](https://github.com/user-attachments/assets/ecd6473d-c884-44af-b710-8f0810f3a851)

If we want something that stands out more than a usual link, we can give it a `class` and add a rule for it in assets/css/global.css that applies a border and/or background.